web/src/features/ApplicationDetail/ApplicationDetailPage.module.css

```
@@ -58,59 +58,55 @@
   margin: 0 0 var(--tc-space-sm) 0;
 }
 
+/* ---------- Status stamp: outlined, uppercase, icon + label always ---------- */
 .badge {
   display: inline-flex;
   align-items: center;
   gap: var(--tc-space-xs);
   padding: var(--tc-space-xs) var(--tc-space-sm);
-  border-radius: var(--tc-radius-full);
+  border: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)